sources/Launcher/DYNRobustnessAnalysisLauncher.cpp

```diff
@@ -382,6 +382,11 @@ RobustnessAnalysisLauncher::simulate(const boost::shared_ptr<DYN::Simulation>& s
     } catch (const DYN::MessageError& m) {
       std::cerr << m.what() << std::endl;
       Trace::error() << m.what() << Trace::endline;
+
+      std::string m_str(m.what());
+      std::replace(m_str.begin(), m_str.end(), '\n', ' ');
+      result.setCriticalTimeMessageError(m_str);
```

Comment:
wrong name for this method. Should be "setSimulationErrorMessage" or something. Moreover why is it only set in the case of MessageError and not DYN::Error for example?

Reply:
I think this is not a wrong name. Here we want to catch the error message to put it later  in XML result specially for critical time.